lang_portal_backend/scripts/seed_sentence_builder.py

```
@@ -0,0 +1,485 @@
+            "pattern": "subject adverb verb",
+            "description": "Subject with adverb modifying the verb (without full stop)",
+            "example": "میں تیزی سے جاتا/جاتی ہوں (I quickly go)",
+            "difficulty_level": 2
+        }
+    ]
+    
+    # Clear existing embeddings in ChromaDB
+    try:
+        sentence_collection.delete(ids=[f"pattern_{i+1}" for i in range(len(patterns) * 2)])  # Account for more patterns
+    except:
+        pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)